src/Uno.UI/UI/Xaml/FontFamily.Loader.wasm.cs

```diff
@@ -0,0 +1,158 @@
+﻿#nullable enable
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Uno.Foundation;
+using Uno.UI.DataBinding;
+using Windows.Storage.Helpers;
+
+namespace Windows.UI.Xaml.Media
+{
+	partial class FontFamily
+	{
+		private class FontLoader
+		{
+			private static readonly Dictionary<FontFamily, FontLoader> _loaders = new();
+
+			internal static FontLoader GetLoaderForFontFamily(FontFamily forFamily)
+			{
+				if (_loaders.TryGetValue(forFamily, out var loader))
+				{
+					// There is already a loader for this font family
+					return loader;
+				}
+
+				loader = new FontLoader(forFamily);
+				_loaders.Add(forFamily, loader);
```

Comment:
Any concurrency concerns here? 
I think this is problematic if multiple threads execute this line, but I'm not sure if this can happen.

Reply:
This cannot happen in this context, the code is always called from the UI thread.